providers/agent/mcorpc/audit/choria_unix_test.go

```diff
@@ -12,16 +12,16 @@ import (
 	"strconv"
 	"syscall"
 
-	"github.com/onsi/gomega"
+	. "github.com/onsi/gomega"
```

Comment:
For my own curiosity, how did you get staticcheck to not dislike this dot import?

Reply:
I believe staticcheck only cares about non test files for this check.  A file called `x_test_unix.go` is not a test but `x_unit_test.go` is a test 

Thats my theory anyway - renaming files is first thing I tried